### DISCUSSION TITLE
Customer phone number in checkout not saved

### DISCUSSION THREAD

**Discussion by Scippy** [HotcakesCommerce/hotcakes-commerce-core#448] [General]
I've HCC 3.2.3 and I've no possibility to update it to the latest version due to compatibility problems with the DNN version.
Everything works for me but I have a problem on retrieving the customer phone number.
In checkout process after the customer enter his phone number through _CheckoutAddress.cshtml form it is not visible anywhere.
I've searched everywhere and it just seems like it's nowhere to be seen.
Is it possible that it is not saved?
Where can I find it directly in the database to check if it is saved or not?
In _CheckoutAddress.cshtml it is saved with the following lines of code:

`<div class="form-group">
            <label class="col-sm-4 control-label" for="@(Model.Prefix + "phone")">@Localization.GetString("Phone"): </label>
            <div class="col-sm-offset-0 col-sm-12">
                <input type="text" id="@(Model.Prefix + "phone")" name="@(Model.Prefix + "phone")"
                    value="@Model.Address.Phone" class="form-control @Model.IsErr("phone")" placeholder="@Localization.GetString("phPhone")" />
            </div>
        </div>`

is there any mistake there?
Thanks for your help!

**Comment by WillStrohl** [comment]
Why are you not able to upgrade Hotcakes Commerce?  What compatibility issue are you aware of with DNN?  We're not currently aware of any compatibility issues at this time.  Also, 3.2.3 is quite old.  We recently released 3.6 and are looking at moving directly to 3.7 very soon.
